src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/Term.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis
+
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Term as IntegrationApiTerm
+
+data class Term(
+  val years: Int? = null,
+  val months: Int? = null,
+  val weeks: Int? = null,
+  val days: Int? = null,
+) {
+  fun toTerm(): IntegrationApiTerm = IntegrationApiTerm(
+    years = if (this.years != 0) this.years else null,
+    months = if (this.months != 0) this.months else null,
+    weeks = if (this.weeks != 0) this.weeks else null,
+    days = if (this.days != 0) this.days else null,
```

Comment:
Noticing that this function returns else null ... is the best decision we can make?
As pointed out earlier, the data coming from Nomis simply doesn't include these fields if they were null at Nomis end... can we implement this instead?
They achieve it with annotations so how would we achieve it?